Example/Messaging/App/watchOS/SampleWatchApp/SampleWatchApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,741 @@
+// !$*UTF8*$!
```

Comment:
May want to `pod deintegrate` to make this easier to manage

Reply:
hmm this is after I ran "pod deintegrate'.

Reply:
Oh looks like you ran with Xcode open on the project and it left the Pods references in it. You can delete from the prom the source or the Pods references in the Xcode window

Reply:
okay I think I clear it, Can you still see pod references?